### DISCUSSION TITLE
How to host this on render?

### DISCUSSION THREAD

**Discussion by ngmisl** [Thomasbehan/LesNet#25] [Q&A] [answered]
Is there an easy setup to host this like you do on render? If yes, how?

thanks, and this is a great project!

**Comment by Thomasbehan** [comment, marked as answer]
Hey,

Here's a step-by-step guide to deploy the app on Render:

1. Go to your dashboard on Render.com.
2. Click on the New button at the top right of the dashboard.
3. Select Web Service from the dropdown menu.
4. You'll see an option to provide a repository URL. Here, paste the URL of the SkinVestigatorAI repository: https://github.com/Thomasbehan/SkinVestigatorAI. (This can be any pubic repo)
6. After pasting the URL, click on the Create Web Service button at the bottom of the form.
7. The Render platform will then automatically build and deploy the application.

Glad to hear that you're finding the SkinVestigatorAI project valuable. Good luck with your deployment​

**Reply by ngmisl** [reply]
Thank you! Any chance you'll change to a more open license? I'd like to change the interface, but GPL3 doesn't allow this

**Reply by Thomasbehan** [reply]
Unlikely to change the license sadly, but feel free to make a PR with any UI updates you might want.
